Mathlib/Tactic/CDotLinter.lean

```diff
@@ -0,0 +1,73 @@
+/-
+Copyright (c) 2024 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import Lean.Elab.Command
+import Lean.Linter.Util
+
+/-!
+#  The "generic" linter
+
+The "generic" linter takes as input a function
+`unwanted : Syntax → Array Syntax` that returns the syntax nodes of an input syntax that
+should be flagged.
+
+
+TODO:
```

Comment:
For the cdot linter, did any of these pose an issue? (I don't recall seeing such, but I might be wrong.)
Otherwise, "Future possibilities" seems like another option - to clarify this is not blocking this linter from landing.

Reply:
No, these were ideas that I wanted to jot down.  I like "Future", I used "Future extensions".